diff -ur rpmorphan-1.19.orig/Makefile rpmorphan-1.19/Makefile
--- rpmorphan-1.19.orig/Makefile	2018-09-14 17:45:29.000000000 +0200
+++ rpmorphan-1.19/Makefile	2021-04-08 10:03:49.757653188 +0200
@@ -8,7 +8,7 @@
 VERSION = $(shell grep Version rpmorphan.lsm | awk '{print $$2}')
 
 BINDIR = $(DESTDIR)/usr/bin
-LIBDIR = $(DESTDIR)/usr/lib/rpmorphan/
+LIBDIR = $(DESTDIR)/usr/share/rpmorphan/
 LOGETC = $(DESTDIR)/etc/
 LOGROT = $(LOGETC)/logrotate.d/
 MANDIR = $(DESTDIR)/usr/share/man
@@ -19,7 +19,7 @@
 # for keep file 
 DATADIR = $(DESTDIR)/var/lib/rpmorphan
 # locales
-TARGET_LOCALE = $(DESTDIR)/usr/lib/rpmorphan/locale
+TARGET_LOCALE = $(DESTDIR)/usr/share/rpmorphan/locale
 
 # log
 LOGDIR  = $(DESTDIR)/var/log
diff -ur rpmorphan-1.19.orig/rpmdep.pl rpmorphan-1.19/rpmdep.pl
--- rpmorphan-1.19.orig/rpmdep.pl	2018-09-14 17:45:29.000000000 +0200
+++ rpmorphan-1.19/rpmdep.pl	2021-04-08 10:03:49.757653188 +0200
@@ -30,7 +30,7 @@
 
 # library
 push @INC, dirname($PROGRAM_NAME);    # dev, tgz
-push @INC, '/usr/lib/rpmorphan/';     # prod rpm
+push @INC, '/usr/share/rpmorphan/';     # prod rpm
 
 ## no critic (RequireBarewordIncludes)
 require 'rpmorphan-lib.pl'
diff -ur rpmorphan-1.19.orig/rpmduplicates.pl rpmorphan-1.19/rpmduplicates.pl
--- rpmorphan-1.19.orig/rpmduplicates.pl	2018-09-14 17:45:29.000000000 +0200
+++ rpmorphan-1.19/rpmduplicates.pl	2021-04-08 10:03:49.757653188 +0200
@@ -28,7 +28,7 @@
 use File::Basename;
 
 ## library is not necessary in current code
-#use lib '/usr/lib/rpmorphan/';
+#use lib '/usr/share/rpmorphan/';
 #push @INC, dirname($PROGRAM_NAME);
 #
 ### no critic (RequireBarewordIncludes)
diff -ur rpmorphan-1.19.orig/rpmextra.pl rpmorphan-1.19/rpmextra.pl
--- rpmorphan-1.19.orig/rpmextra.pl	2018-09-14 17:45:29.000000000 +0200
+++ rpmorphan-1.19/rpmextra.pl	2021-04-08 10:03:49.757653188 +0200
@@ -31,7 +31,7 @@
 
 # library
 push @INC, dirname($PROGRAM_NAME);    # dev, tgz
-push @INC, '/usr/lib/rpmorphan/';     # prod rpm
+push @INC, '/usr/share/rpmorphan/';     # prod rpm
 
 ## no critic (RequireBarewordIncludes)
 require 'rpmorphan-lib.pl'
diff -ur rpmorphan-1.19.orig/rpmorphan.pl rpmorphan-1.19/rpmorphan.pl
--- rpmorphan-1.19.orig/rpmorphan.pl	2018-09-14 17:45:29.000000000 +0200
+++ rpmorphan-1.19/rpmorphan.pl	2021-04-08 10:03:49.757653188 +0200
@@ -29,7 +29,7 @@
 
 # library
 push @INC, dirname($PROGRAM_NAME);    # dev, tgz
-push @INC, '/usr/lib/rpmorphan/';     # prod rpm
+push @INC, '/usr/share/rpmorphan/';     # prod rpm
 
 ## no critic (RequireBarewordIncludes)
 require 'rpmorphan-lib.pl'
diff -ur rpmorphan-1.19.orig/rpmusage.pl rpmorphan-1.19/rpmusage.pl
--- rpmorphan-1.19.orig/rpmusage.pl	2018-09-14 17:45:29.000000000 +0200
+++ rpmorphan-1.19/rpmusage.pl	2021-04-08 10:03:49.757653188 +0200
@@ -28,7 +28,7 @@
 
 # library
 push @INC, dirname($PROGRAM_NAME);    # dev, tgz
-push @INC, '/usr/lib/rpmorphan/';     # prod rpm
+push @INC, '/usr/share/rpmorphan/';     # prod rpm
 
 ## no critic (RequireBarewordIncludes)
 require 'rpmorphan-lib.pl'
